public/data/izzarzn.json

```diff
@@ -13,6 +13,34 @@
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/izzarzn",
       "icon": "twitter"
+    },
+    {
+      "name": "Check out my website",
+      "url": "https://izzarzn.github.io",
```

Comment:
There is a broken link, check whether there is a typo, or you've placed the correct URL!!

Please update it with the correct one!! 🛠

![image](https://user-images.githubusercontent.com/88786642/189509987-aeb9ccd1-1ca0-4e92-821d-120ca3c2b632.png)



__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant, Not maintainable